src/allmydata/_auto_deps.py

```diff
@@ -39,7 +39,6 @@
     ('pycparser',        'pycparser'),
     ('PyYAML',           'yaml'),
     ('magic-wormhole',   'wormhole'),
-    ('setuptools',       'setuptools'),
```

Comment:
"That includes setuptools and pkg_resources." can be deleted from the comment on line 16.

Reply:
Also I'm pretty sure lines 5 to 12 inclusive are not needed.